src/core/Alert/InlineAlert/InlineAlert.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component, forwardRef } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlDiv, HtmlDivWithRef } from '../../../reset';
+import { Icon } from '../../../core/Icon/Icon';
+import {
+  BaseAlertProps,
+  alertClassNames,
+  baseClassName,
+} from '../BaseAlert/BaseAlert';
+import { AutoId } from '../../utils/AutoId/AutoId';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../theme';
+import { baseStyles } from './InlineAlert.baseStyles';
+
+export interface InlineAlertProps extends BaseAlertProps {
+  /** Label for the alert */
+  labelText?: string;
```

Comment:
This might be unwanted repetition, as `labelText` prop seems already in the `BaseAlertProps`.

Reply:
I looked this closer and it seems that **InlineAlert** is the only one using the `labelText` prop. So it should be removed from the **BaseAlertProps** instead.